coderd/notifications/render/gotmpl_test.go

```diff
@@ -0,0 +1,59 @@
+package render_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/v2/coderd/notifications/render"
+
```

Comment:
nit: empty line

Reply:
I'm relying on `make/fmt` to do the right thing.